include/nbl/video/utilities/IGPUObjectFromAssetConverter.h

```diff
@@ -1089,16 +1022,34 @@ auto IGPUObjectFromAssetConverter::create(const asset::ICPUImage** const _begin,
         IGPUImage::SCreationParams params = {};
         params = cpuimg->getCreationParameters();
         params.initialLayout = asset::IImage::EL_UNDEFINED;
+        
+        IPhysicalDevice::SImageFormatPromotionRequest promotionRequest = {};
+        promotionRequest.originalFormat = params.format;
+        promotionRequest.usages = {};
```

Comment:
I think you can prime it at least with the sampling & linear filtering (if format capable of linear filtering) usage